open-api/rest-catalog-open-api.yaml

```diff
@@ -2068,6 +2165,145 @@ components:
+          $ref: '#/components/schemas/ContentFile'
+        partition:
+          type: object
+          additionalProperties:
+            type: string
+        size-bytes:
+          type: number
+        start:
+          type: number
+        length:
+          type: number
+        estimated-rows-count:
+          type: number
+        delete-files:
+          type: array
+          items:
+            $ref: '#/components/schemas/ContentFile'
+        schema:
+          $ref: '#/components/schemas/Schema'
+        spec:
+          $ref: '#/components/schemas/PartitionSpec'
+        residual-filter:
+          $ref: '#/components/schemas/Expression'
+
+    ContentFile:
+      type: object
+      required:
+        - file-path
+      properties:
+        spec-id:
+          type: string
+        content:
+          $ref: '#/components/schemas/FileContent'
+        file-path:
+          type: string
+        file-format:
+          type: string
+        partition:
+          type: object
+          additionalProperties:
+            type: string
+        record-count:
+          type: number
+        file-size-in-bytes:
+          type: number
+        column-sizes:
+          type: object
+          additionalProperties:
+            type: object
+            properties:
+              keys:
+                type: integer
+              values:
+                type: number
+        value-counts:
+          type: object
+          additionalProperties:
+            type: object
+            properties:
+              keys:
+                type: integer
+              values:
+                type: number
+        null-value-counts:
+          type: object
+          additionalProperties:
+            type: object
+            properties:
+              keys:
+                type: integer
+              values:
+                type: number
+        nan-value-counts:
+          type: object
+          additionalProperties:
+            type: object
+            properties:
+              keys:
+                type: integer
+              values:
+                type: number
+        lower-bounds:
+          type: object
+          additionalProperties:
+            type: object
+            properties:
+              keys:
+                type: integer
+              values:
+                type: string
+                format: binary
+        upper-bounds:
+          type: object
+          additionalProperties:
+            type: object
+            properties:
+              keys:
+                type: integer
+              values:
+                type: string
+                format: binary
+        key-metadata:
+          type: string
+          format: binary
+        split-offsets:
+          type: array
+          items:
+            type: number
+        equality-ids:
+          type: array
+          items:
+            type: integer
+        sort-order-id:
+          type: integer
+        data-sequence-number:
```

Comment:
I don't think data-sequence-number and file-sequence-number are in the ContentFileParser, why are they added here?